modules/ROOT/pages/design-create-publish-api-visual-editor.adoc

```diff
@@ -5,7 +5,7 @@ endif::[]
 
 :imagesdir: ../assets/images
 
-You can use API Designer's visual API editor to scaffold RAML 1.0 or OAS 2.0 (JSON) API specifications by populating language-neutral forms.
+You can use API Designer's visual API editor to scaffold RAML 1.0, or OAS 2.0 or 3.0 (JSON), API specifications by populating language-neutral forms.
```

Comment:
Hmm...this one was hard for me to parse. Please look at it again. 
I THINK you mean:
```suggestion
You can use API Designer's visual API editor to scaffold RAML 1.0 or OAS 2.0 or 3.0 (JSON) API specifications by populating language-neutral forms.
```
You can ...scaffold
(RAML
or OAS)
API specs by...